content/docs/capabilities/authorization.mdx

```diff
@@ -44,14 +44,11 @@ Pomerium's approach to authorization is continuous and context-aware, integratin
 
 ## Where Policies Live
 
-1. **Routes**  
-   Policies can be attached to each route, controlling who and what can access the upstream service.
+1. **Routes** Policies can be attached to each route, controlling who and what can access the upstream service.
```

Comment:
Can you put the line break back? This might be a prettier issue, but ideally it is on the next line or we put a hypen in, because this change from newline to space causes it display differently on the page.